### ISSUE TITLE
Italics

### ISSUE THREAD

**Issue by jmuyskens** [adobe-fonts/source-serif-pro#2]
Are italic versions in the works?

**Comment by PeterBocan** [comment]
Any news on that? It's almost a year ago, guys...

**Comment by pauldhunt** [comment]
There is a ton of news, just not an official version yet. You can follow all the latest changes here:
https://github.com/pauldhunt/SourceCodePro/tree/PaulVersionCurrent

**Comment by adrientetar** [comment]
@pauldhunt I think you're confusing repos, this is Source Serif Pro here.:)

**Comment by pauldhunt** [comment]
d��oh! apparently i have SCP italics on the brain, which have also been taking plenty of time. font development can be a slow process, especially if you have multiple projects that require attention.

**Comment by miguelsousa** [comment]
@PeterBocan it's probably going to be another year until the italics materialize. @frankrolf has been sketching them out, but he has higher priority projects in his queue.

**Comment by PeterBocan** [comment]
Okay, thank you, @miguelsousa :)

**Comment by pstonier** [comment]
Any update on where these stand? I'm excited for how this is progressing.

**Comment by frankrolf** [comment]
Still in active development, however without a specific deadline.

**Comment by pstonier** [comment]
ok. Thank you.

**Comment by andya9** [comment]
Thank’you for the beautiful font!
(Btw, you can leave an issue open but prevent adding more comments to it by locking it, see here https://github.com/blog/1847-locking-conversations)

**Comment by frankrolf** [comment]
Thanks @andya9! Sorry for being grouchy here originally, still working on the Italics! :)

**Comment by andya9** [comment]
No probs, thank’you again :smile:

**Comment by mbutterick** [comment]
You got [monkeypatched](https://doc.rust-lang.org/nightly/reference.html#character-escapes)!

**Comment by miguelsousa** [comment]
@mbutterick Utopia Italic derivative? Seems to work well together at first glance.

**Comment by adrientetar** [comment]
> Utopia Italic derivative?

Yes, it's called Heuristica. Originally we were using it in the upright style too but people weren't very satisfied with it and I found Source Serif Pro to be a better fit for the documentation. 
But it doesn't have uprights.

Anyway. I think it works fairly well here.

**Comment by frankrolf** [comment]
It’s nice to see how lacking italics inspire typographic creativity. 
Eventually the Rust documentation will be able to use Source Serif throughout, but not just yet.

**Comment by dmoonfire** [comment]
I went another approach of using [the lighter weight](https://fedran.com/sand-and-ash/chapter-01/) (in the epigraph and the line that starts with "As the sun touched the horizon, the *dépa* turned sharply") for italics. I was originally planning on switching to italics when they were available, but using the light for italics has been growing on me. I'm doing the same with Source Serif Pro also since they are on the same page on that site.

**Comment by frankrolf** [comment]
@dmoonfire Great! There are many ways for expressing typographic hierarchy other than using Italics. I am not apologizing here, just making a point for people saying Source Serif is “unusable” for its lack of Italics. They will come eventually!

**Comment by mbutterick** [comment]
### ☹ U+2639 SAD ADOBE FACE

**Comment by frankrolf** [comment]
### :trollface: U+666G TROLL FACE

**Comment by LandonSchropp** [comment]
+1 :smile:

**Comment by KingScooty** [comment]
_1

**Comment by philipzae** [comment]
+1 - LibreOffice wishes to bundle the font with its installation, as it already bundles Source Sans, but is awaiting italics to be completed. https://bugs.documentfoundation.org/show_bug.cgi?id=79022

@frankrolf is there some means to assisting in its completion?

**Comment by frankrolf** [comment]
That is very nice to hear!
Encouragements like this are good arguments for internal politics, so in a way, you are assisting already! Other than that, I think I just need a bit more time.
Thank you!


from a two-dimensional keyboard

**Comment by nathanjweber** [comment]
+1 

In a way, restriction has bred creativity on our team but having the flexibility of using italics in some areas would be nice. It's especially necessary for our work since italics mean something specific to our users as a way of differentiating references and cites. Hope it comes to fruition soon!

**Comment by mg-13** [comment]
Why am I able to set `font-style: italic;` in CSS when viewing the font on the web (e.g. in [typekit](https://typekit.com/fonts/source-serif-pro) )?
Does the browser just tilt this over? Works in Firefox and IE (no webkit currently installed)...

**Comment by mrzool** [comment]
@mg-13  That's exactly what browsers do, it's called faux italic. 

> If text is styled as bold or italic and the typeface family does not include a bold or italic font, browsers will compensate by trying to create bold and italic styles themselves. ([via](http://alistapart.com/article/say-no-to-faux-bold))

**Comment by KurtPfeifle** [comment]
Has development of this font stalled? Latest commit (other than a small update to the README 6 months ago) was more than a year ago: *"commit 03b7185 on Jan 16, 2015"*.

Should I let go all hope for a real italic version?

**Comment by miguelsousa** [comment]
@KurtPfeifle Development on this project is very much alive, and coincidentally @frankrolf and I just finished discussing plans for a new build of the upright fonts coming out within the next couple of months or so.
Progress on the design of the italics has also happened in the meantime, but it's still too early to tell when those will be released.

**Comment by ShikiSuen** [comment]
@KurtPfeifle It's about time to give up Source Serif Pro unless you use alternative formatting in lieu of italics (e.g. dotted underline, etc). I recommend Adobe Utopia at this moment to cope with necessities of italics.

**Comment by frankrolf** [comment]
Tough words. 
I can assure you all that the lack of Italics has nothing to do with laziness or lack of dedication. However, there are internal priorities that take precedence. Here in this repository, of course it looks like the project is “dead”, but progress is being made every day. There are very good reasons for this progress not being public.

Being critical toward my work, I would rather release no Italics than half-baked ones, and I encourage you all to either find alternate typefaces, or – if you _have_ to use Source Serif, use different ways for typographic emphasis.

**Comment by KurtPfeifle** [comment]
@miguelsousa: Thanks a lot for the update. Good to know that we've need to be a bit more patient.

**Comment by kenlunde** [comment]
@ShikiSuen: As an observer of this thread, I would like to state that you are being overly and inappropriately critical toward my highly-talented colleague, @frankrolf. As he stated in his reply, this project is not his only responsibility, and while other font developers may choose to release something that is half-baked, our team does not.

I would also like to point out that all of the projects in this repository are _very_ important for our team, and we never take a "cut the umbilical cord" attitude toward them. Some projects require more attention than others.

In other words, please continue to be patient. 😉

**Comment by ShikiSuen** [comment]
cc @kenlunde @miguelsousa @frankrolf >

I apologize for your bad feeling led from my previous reply. Meanwhile, I need to address the following:

1. I didn't and do never want to make anyone here suffer.

2. I always think it's a terrible idea to leave people here long-time waiting and murmuring for something which was "supposed to be published" but not published for a really long time. This led to my previous reply persuading people to temporarily walk away and forget this project.

3. Even though I said "always better to give something up than die waiting", it is people "die waiting" but not this gorgeous project (I still think Source Serif Pro is the best serif font in recent years, as what I addressed on Twitter before: https://twitter.com/ShikiSuen/status/709789640425807872). I think people "die waiting" is because they could use these waiting time to do something more valuable in lieu of pushing developers into the following situation.

4. "Ants onto the oven" is a daily metaphor borrowed from Mandarin, describing a situation like "people gets pushed and feel too upset to concentrate on what they are working on" but definitely NOT related to any mean of laziness and lack of dedication. My apologies that I was supposed to use daily English metaphor in this thread but my English education has lack of that.

**Comment by ShikiSuen** [comment]
// This is just a courtesy notification to email subscribers that my previous reply has been updated.

**Comment by eyerhyme** [comment]
Thank you Adobe, @frankrolf and your hard working team for giving such a generous gift to the world as these high quality open source typefaces (sans and serif). They have been widely praised by designers and the type design community. 

I am looking forward to italics like everyone here, but feel rather humble to be placing any demands, because it is _really_ a gift they are giving; and I am not financially supporting all their hard work in any way. 

I am not a type designer, but all accounts are that good work like this takes a tremendous amount of time and artistic effort...many years often. I encourage all here to be patient and trust what the designers have said: they are working on it. I hope the design team feels free to take all the time they need for a quality product that they are proud of.

**Comment by PeterBocan** [comment]
"Good things come to those who wait." 
I have no problem waiting another year.

**Comment by ShikiSuen** [comment]
I am going to remove my previous disputing comment because there are still down-votes coming even though I wrote full explanations and embedded it. There are always people like @fitojb who NEVER read explanations but just using down-votes to comfort their eager of entertaining others.

**Comment by kenlunde** [comment]
@ShikiSuen: Please accept this constructive advise.

You aren't doing yourself any favors by removing or seriously editing your posts, mainly because it disrupts the discussion flow and removes all-important context, but perhaps more importantly because your [activity](https://github.com/ShikiSuen?tab=activity) is always visible.

We all make mistakes, or write things in the heat of passion that we later regret, and I am as guilty as the next person when it comes to this. Owning up is always the prudent approach for such matters.

_(Also, I suggest that you change your avatar.)_

**Comment by ShikiSuen** [comment]
@kenlunde >

1. The context is not interrupted. I backed up my original reply and embedded it below my explanation to make sure people always see it after they saw my explanation.

2. The entire thing is just my lack of explanation. 

In my original text, nothing is ad hominem to Frank. I admit that I did see Frank suffered, and I feel sorry about that. However, What made him suffer is due to his comprehension towards my original reply.

I said "always better to give something up than die waiting, pushing developers like ants on the oven"; the subject (主語) of this sentence, even though is not written out, is evidently not font developers. The thing pushing developers is people's non-stop waiting and complaining about the delayed new-version. (Hope that I did no grammatical error here.)

Meanwhile, you told me that you and your comrades have mainly commercial projects to work on, and this is evident to me. That's why I initially assumed that the next update, regarding current repo, needs approximately two years.

Also, according to my recent explanation reply, I explained the metaphor with appropriate source (probably not academic from your perspective since it is Wikitonary).

Regretfully, when you send me first reply regarding this incident, you didn't get my idea even though I already explained thoroughly in my initial explanation. You may think I was blaming Frank for the "nature of laziness", but I did not. (If you didn't think in such way, you may not need to state the same thing again in your email)

I could just leave my initial comment there as-is because it has Nothing ad-hominem to Any of people working on this repo. However, Frank did mis-comprehend my idea and left his reply above, and so does your statement. All of such are attracting those passing visitors to leave a downvote without any eager of reading my explanation. No one gets it but just want to throw a downvote on someone so that they could feel it cool like justice.

I know your reason to leave the context as-is. However, for those people who visiting this thread or subscribed, they only want to hear news related to those project but definitely not my explanation. This leads to a fact that it does no help to leave dialogs non-related to this reop. Thus, for the God's sake, please, you @kenlunde and Frank @frankrolf remove yours and I will remove mine later, just let people know that this thread is not related on useless complaints. // If you don't want (plus you still didn't get my idea), I will remove all of my replies above (incl. this one) to divest people's rights to entertain me here by using their non-reasonable downvotes.

**Comment by ShikiSuen** [comment]
Again, I have no responsibility to comfort someone who misunderstood my statements. All of doing such (e.g. my following-up explanations) is to let the people who misunderstood my statements stop suffering anymore just because I hate to see people suffering (especially "related" to what I said). However, your left comments led from your miscomprehension really hurt me constantly since such comments are persuading people to do the same miscomprehension like yours. Please let me know if this is sort of "revenge" since I am aware of a fact that one could only express those feelings he felt.

**Comment by frankrolf** [comment]
Any minute invested in removing and/or editing comments takes away my time from working on the Italics. With that in mind, I would encourage everyone to just move on, and not make this a bigger thing than it is.
I felt the need to respond to the original (now deleted) comment by @ShikiSuen, but I don’t feel the necessity to edit this conversation.

**Comment by pathawks** [comment]
If you’d like to reduce the noise, you can always lock the conversation to contributors only. That way, those of us who have subscribed for updates can continue to recieve only the relevant information.

**Comment by frankrolf** [comment]
You can follow the progress of Source Serif Italic [in my private fork](https://github.com/frankrolf/source-serif-pro/tree/italic/ItalicMasters).

**Comment by mnmiller78** [comment]
Coming soon?

**Comment by MuppetGate** [comment]
Thanks for working in this :-)

**Comment by MuppetGate** [comment]
Er… I meant "working on this".

**Comment by jrmiller82** [comment]
Any idea when italics are going to be added? It's hard to actually use a serif font for any sort of real body text without italics...

**Comment by samboy** [comment]
Is there any way we can help fund the development of italics?

For people who want italics right now, one solution is to use Heuristica italic: https://sourceforge.net/projects/heuristica/

**Comment by sn0wyfall** [comment]
@samboy This works well. Thanks!

**Comment by frankrolf** [comment]
@samboy Thanks for that. It’s a good solution for the time being. There are several reasons why no Italics exist yet, but there are also may ways to get around that.

One thing I don’t understand – why does the FontSquirrel download say:
Heuristica – designed by Andrej Panov? 
Heuristica is Utopia, which has been designed by Robert Slimbach.
(Not questioning Panov’s Cyrillic extension, but critical toward taking over original design credit)

**Comment by frankrolf** [comment]
Regarding this statement:
> a bizarre license Adobe came up with when giving the Utopia font to the TeX User’s group

Utopia was probably one of the first open source fonts. There was no OFL at that point. Perhaps that’s why that license seems bizarre now, and perhaps it should be revisited (if that’s even possible).

**Comment by KyleAMathews** [comment]
Added Heuristica to NPM https://www.npmjs.com/package/typeface-heuristica

**Comment by samboy** [comment]
Since Frank is asking about Utopia’s license, here is the license under which Adobe made Utopia open source, in 2006:

```
Adobe Systems Incorporated ("Adobe") hereby grants to the TeX Users
Group and its members a nonexclusive, royalty-free, perpetual license to
the typeface software for the Utopia Regular, Utopia Italic, Utopia Bold
and Utopia bold Italic typefaces, including Adobe Type 1 font programs
for each style (collectively, the "Software") as set forth below.

Adobe grants the TeX Users Group a license under its copyrights, to use,
reproduce, display and distribute the Software for any purpose and
without fee provided that the following copyright notice appears in all
whole and partial copies of the Software and provided that the following
trademark symbol and attribution appear in all unmodified copies of the
Software:

Copyright 1989, 1991 Adobe Systems Incorporated. All rights reserved.
(alternatively, @1989, 1991 Adobe Systems Incorporated. All rights reserved.)
Utopia(R)
Utopia is either a registered trademark or trademark of Adobe Systems
Incorporated in the United States and/or other countries. Used under
license.

Adobe also grants to the TeX Users Group a license to modify the
Software for any purpose and redistribute such modifications, for any
purpose and royalty-free, provided that the modified Software shall not
use the font name(s) or trademark(s), in whole or in part, unless
explicit written permission is granted by Adobe. This restriction
applies to all references stored in the Software for identification
purposes, such as the font menu name and other font description
fields. The TeX Users Group is also permitted to sublicense, and grant
such sublicensees the right to further sublicense, any or all the
foregoing rights through multiple tiers of distribution. The licenses
granted herein are granted in perpetuity and may not be terminated by
either party unless such termination is based on a breach of the terms
and conditions herein stated.

Adobe retains ownership of the copyright in the Software. The TeX Users
Group agrees that Adobe and its suppliers are the sole and exclusive
owners of all rights, title and interest, including all copyrights,
patents, trademarks, trade names, trade secrets and other intellectual
property rights in the Software. No title or ownership of the Software,
any copies of the Software, or the patent, copyright, trade secret,
trademark, trade name or other proprietary rights contained in the
Software is transferred to the TeX Users Group.

The Adobe trademarks shall not be used in advertising pertaining to the
distribution of the Software without express prior permission from
Adobe. Any such use shall be in accordance with the Adobe trademark
guidelines, available on the Adobe website at
http://www.adobe.com/misc/pdfs/TM GuideforThirdPartiesFinal.pdf.
If any portion of the Software is changed, it cannot be marketed under
Adobe's trademarks unless Adobe, in its sole discretion, approves by a
prior writing the quality of the resulting implementation.

The TeX Users Group shall have the right to evaluate the Software
provided by Adobe.
```

Then the TeX Users Group (TUG) gave all such rights, irrevocably and in perpetuity, to any and all interested parties.

Heuristica added a version of the OFL to the above license.

The original open sourcing of Utopia was done in the 1990-1992 timeframe to be part of the X windows system; this is also when Matthew Carter’s Bitstream Charter font was similarly donated (along with a version of Courier, one from IBM, another from Bitstream). An old Usenet posting summarizes all this: https://groups.google.com/forum/#!original/comp.fonts/ZrDhqYyvUU8/lEqqtgtv12EJ (Usenet, message ID <EPPERLY.93Apr13101845@osnome.che.wisc.edu>), with the original license for giving away Utopia:

```
%   Permission to use, reproduce, display and distribute the listed
%   typefaces is hereby granted, provided that the Adobe Copyright notice
%   appears in all whole and partial copies of the software and that the
%   following trademark symbol and attribution appear in all unmodified
%   copies of the software:
%
%           Copyright (c) 1989 Adobe Systems Incorporated
%           Utopia (R)
%           Utopia is a registered trademark of Adobe Systems Incorporated
%
%   The Adobe typefaces (Type 1 font program, bitmaps and Adobe Font
%   Metric files) donated are:
%
%           Utopia Regular
%           Utopia Italic
%           Utopia Bold
%           Utopia Bold Italic
```

I do not think it was the intention of Adobe nor Bitstream to make these fonts available to anyone, anywhere; this was before the Internet was readily available outside of universities. 

Personally, I am using Bitstream Charter for the time being. Both Charter and Source Serif Pro are based on the letters of Pierre-Simon Fournier; however, Charter has more slab-like serifs because it was designed to be a very compact font — Matthew Carter was using a typeface system with limited space for vector fonts that could only really handle sans typefaces, and designed Charter to work with that system; ironically, by the time Carter finished Charter, the programmers improved the system to allow for bigger typefaces. 

Some Bitstream Charter downloads, since this font for some reason isn’t readily available on Google Fonts: http://practicaltypography.com/charter.html — There’s also the Charter derivative Charis SIL which has a slightly heavier weight than Charter and a greatly expanded character set: http://software.sil.org/charis/

**Comment by rbaylis91** [comment]
Just wanted to chime in and say thanks for all the awesome work on this Frank, and looking forward to seeing Italics added in at some point.

**Comment by jrmiller82** [comment]
Any update on this?

**Comment by inferno986return** [comment]
Indeed, I two would like to know the current status on the italics. :-)

Source Serif Pro is a beautiful typeface, but bold alone only goes so far.

**Comment by frankrolf** [comment]
You can follow the current status of the italic extension in my private fork, as mentioned above:
https://github.com/frankrolf/source-serif-pro

**Comment by samboy** [comment]
@jrmiller82 @inferno986return RTF thread. As I posted above: For people who want italics right now, one solution is to use Heuristica italic: https://sourceforge.net/projects/heuristica/ (If those fonts don’t work, try this link: https://www.fontsquirrel.com/fonts/heuristica )

**Comment by jrmiller82** [comment]
@samboy No need to be hostile. The issue has been open for literally years, the thread doesn't really answer the question (until Frank's reply earlier today). Note: my request about an update was 2 months ago...

**Comment by samboy** [comment]
For people who want to get a sense of the hard work Frank is doing on the Italics, I just whipped out Fontforge and converted the current .ufo file in to a True Type italic font (with the required name change): https://github.com/samboy/NotAtAllOfficialFont I’m sure the final product will look better, but this lets people see how it’s coming along (no, I’m not going to do any weights except the regular weight — if you want more weights, download and install the free Fontforge, which will happily convert .ufo files in to .ttf files. Extra points if you make a GitHub repo with the converted .ttf files.)

In terms of how this looks, I personally prefer how “k” looks in Bitstream Charter italic. The upper left stroke in “z” is a little striking, but I’m used to how Bitstream Charter Italic makes the upper stroke of “z” be a mirror of the lower stroke, and I think the way Source Serif Italic handles it is an interesting, if slightly jarring, contrast for me.

I’m not going to apologize for my hostility. A lot of people have asked for Italics over the years multiple times.

While Italics are not available yet, one can either make unofficial Italic fonts from Frank’s UFO files with Fontforge, or use Heuristica/Utopia (or even Bitstream Charter at https://practicaltypography.com/charter.html) for Italics. Or, get a commercial font like one of Adobe’s many offerings, or something like https://practicaltypography.com/equity.html which includes webfont conversion permission in its font license.

**Comment by jlarmstrongiv** [comment]
I’m not sure not really sure where to post this, so if it should be moved, just let me know.

First of all, thank you for creating Source Serif Pro!  I love the font, and appreciate the hard work you all put into it 👍 

I was checking out @frankrolf’s italics fork ([https://github.com/frankrolf/source-serif-pro](https://github.com/frankrolf/source-serif-pro)), and installed the tools to build the additional weights and italics drafts.  

So far so good!  The only thing that really jumps out at me is where crossbar of the A overlaps the stems (hopefully that’s the right type terminology; see the screenshot below).  It would not surprise me if that’s so you can adjust the crossbar later if you decide to.  However, I was wondering whether that can go ahead and be merged together in the fork, or if that’s just something I should fix on my own by learning enough font software to be dangerous and awaiting future iterations.  

![image](https://user-images.githubusercontent.com/20903247/37742326-8faaa99c-2d3b-11e8-9aa3-d5a444e315c1.png)

I would make an issue on the other repo, but they are not enabled.  I also thought that a pull request without understanding your build tools wouldn’t really be all that helpful.  So yeah.  Anyway, keep up the great work! 🙂

**Comment by frankrolf** [comment]
@jlarmstrongiv Thanks for your concern, as you correctly guessed, drawing characters with overlaps is normal practice, they will be removed in the final release.
You will find overlaps in many more characters than A, and they may serve as a reminder that the fonts are unfinished.
When building the final fonts, we use `checkoutlinesufo` to remove such overlaps.

**Comment by inferno986return** [comment]
It's cool to learn more about font development. I look forward to the day
Source Serif Pro Italic goes live, may even be a champagne moment.

**Comment by frankrolf** [comment]
The first public version of Italic styles supporting the AL-3 character set was released today:
https://github.com/adobe-fonts/source-serif-pro/releases/tag/2.007R-ro/1.007R-it
https://blog.typekit.com/2018/08/16/source-serif-italics/

**Comment by inferno986return** [comment]
This is very exciting, I look forward to trying them out. :-D
If all goes to plan we get them in the next LibreOffice release.

**Comment by mnmiller78** [comment]
This is wonderful. Thank you so very much for your work bringing this to
fruition for all to use. It’s a fantastic font. And I will aim to replace
TNR with it wherever I can.

All the best,
Matt Miller

**Comment by samboy** [comment]
Thank you very much for making these italics available. They are very much worth the wait!

**Comment by miguelsousa** [comment]
I encourage everyone interested in getting some insight on the development process of the Italic to read Frank's blog post https://blog.typekit.com/2018/08/16/source-serif-italics/

**Comment by jrmiller82** [comment]
Awesome! Thank you! Now for true small caps!

**Comment by samboy** [comment]
@jrmiller82 Show them the money! The reason why Source Sans Pro has small caps is because Logos Bible Software paid to make it happen. Reference: https://blog.typekit.com/2012/11/02/source-sans-pro-adoption-and-development-to-date/

**Comment by jrmiller82** [comment]
Apparently it already has small caps; I just don't know where to download the small caps / make them work with Libre Office, Word, LaTeX etc... dunno why you can't download them as a separate font face.

**Comment by miguelsousa** [comment]
@jrmiller82 making separate Small Cap fonts requires additional work that we rather spend in something more fruitful. Please use #33 if you or anyone else wants to keep discussing this topic.
